frontend/src/views/Pages/AccountOverview/GddTable.vue

```diff
@@ -1,104 +1,68 @@
-              <b-list-group-item>
-                <b-badge class="mr-4" variant="primary" pill>gdd</b-badge>
-                {{ item.balance }}
-              </b-list-group-item>
-              <b-list-group-item>
-                <b-badge class="mr-4" variant="primary" pill>{{ $t('form.memo') }}</b-badge>
-                {{ item.memo }}
-              </b-list-group-item>
-            </b-list-group>
-            <b-button v-b-toggle="'collapse-1-inner' + item.transaction_id" variant="secondary">
-              {{ $t('transaction.more') }}
-            </b-button>
-            <b-collapse :id="'collapse-1-inner' + item.transaction_id" class="mt-2">
-              <b-card>{{ item }}</b-card>
-            </b-collapse>
-          </b-card>
-        </b-collapse>
+    <b-list-group
+      horizontal="sm"
+      class="w-100 justify-content-between align-content-space-around"
+      v-for="item in transactions.slice(0, max)"
+      :key="item.id"
+      style="background-color: #ebebeba3 !important"
+    >
+      <b-list-group-item>
+        <b-icon
+          v-if="item.type === 'send'"
+          icon="arrow-left-circle"
+          class="m-1 text-danger"
+          font-scale="2"
+          style="color: red"
+        ></b-icon>
+        <b-icon
+          v-else-if="item.type === 'receive'"
+          icon="arrow-right-circle"
+          class="m-1"
+          font-scale="2"
+          style="color: green"
+        ></b-icon>
+        <b-icon
+          v-else-if="item.type === 'creation'"
+          icon="gift"
+          class="m-1"
+          font-scale="2"
+          style="color: orange"
+        ></b-icon>
+        <b-icon
+          v-else
+          icon="droplet-half"
+          class="m-1"
+          font-scale="2"
+          style="color: orange"
+        ></b-icon>
+      </b-list-group-item>
+      <b-list-group-item class="w-100 align-items-start pt-4">
+        <b class="">
+          <span v-if="item.type === 'receive' || item.type === 'creation'">+</span>
+          <span v-else>-</span>
+          {{ $n(item.balance) }}
+        </b>
       </b-list-group-item>
-      <b-list-group-item v-show="this.$route.path == '/overview'">
-        <b-alert v-if="transactions.length === 0" show variant="secondary">
-          <span class="alert-text">{{ $t('transaction.nullTransactions') }}</span>
-        </b-alert>
-        <router-link
-          v-else-if="transactions.length > 5"
-          to="/transactions"
-          v-html="$t('transaction.show_all', { count: count })"
-        ></router-link>
+      <b-list-group-item class="w-100 align-items-start">
+        <b class="text-muted">{{ item.name }}</b>
+        <div>{{ item.memo }}</div>
+      </b-list-group-item>
+      <b-list-group-item class="w-100 align-items-start">
+        {{ $moment(item.date).format('DD.MM.YYYY - HH:mm:ss') }}
       </b-list-group-item>
     </b-list-group>
+
+    <b-list-group-item v-show="this.$route.path == '/overview'">
```

Comment:
this looks fishy

Reply:
ist im aktuellen master schon nicht mehr drin

Reply:
Ja aber dieser PR fügt es ja hinzu?!